product_docs/docs/transporter/ba_preview/getting_started/prepare_schema.mdx

```diff
@@ -2,4 +2,16 @@
 title: "Preparing schema"
 ---
 
-Use [EDB Migration Portal](https://www.enterprisedb.com/docs/migration_portal/latest/) to assess Oracle database sources for schema compatibility before starting the data migration process. Address incompatible schemas or exclude them from the migration and migrate the schema to the new database **without applying constraints**.
\ No newline at end of file
+Use [EDB Migration Portal to assess Oracle database sources for schema compatibility](https://www.enterprisedb.com/docs/migration_portal/latest/04_mp_migrating_database/02_mp_schema_assessment/#mp_schema_assessment) before starting the data migration process.
+
+In the Migration Portal web application, [address incompatible DDL objects and schemas](https://www.enterprisedb.com/docs/migration_portal/latest/03_mp_using_portal/03_mp_quick_start/) or exclude them from the migration and migrate the schema to the new database.
+
+If you have [constraints](https://www.postgresql.org/docs/current/ddl-constraints.html) in place: 
+
+## Primary Key and Unique constraints
+
+If you have **Primary Key** and **Unique** constraints in your database, ensure you include these constraints when you migrate the schemas to the target Postgres database. Also ensure you create equivalent database tables and constraints. This ensures that the migration can validate and carry them over to the target database and doesn't present any errors.
```

Comment:
In the Migration Portal, the customer can enter a source DDL file with all the schemas from the source database. The customer can then alter/adapt the queries with the help of the portal. They can then export the "fixed" DDL file and import it to their database. 

And here is where it gets a bit blurry to me. 

The fact that the customer must create the database tables and PK & Unique constraints before the migration takes place must be done: 

a. As part of the DDL assessment, import and export process through the Mig Portal? Meaning that in the web UI, the customer would have to exclude all constraints except PK and Unique? 

or 

b. Is the creation of tables and constraints a separate, additional step not related to the schema and DDL import.